phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/unity/main.less

```diff
@@ -440,6 +440,13 @@
     -fx-background-position: left;
 }
 
+.detailsPane {
+    .closeIcon {
+        -fx-border-width: 0;
+        -fx-border-style: none;
```

Comment:
I'll move the `no border` definition to the default theme...
This will reduce the need to override it in the other themes (except the breeze dark theme)